src/components/calcite-dropdown/calcite-dropdown.scss

```diff
@@ -7,7 +7,7 @@
  */
 
 :host {
-  @apply inline-flex flex-initial;
+  @apply inline-flex flex-initial relative;
```

Comment:
Looks good, I only wonder if we should add another container internally to apply this so that it can't be messed with by the end user. Something to contain both the trigger container and the dropdown menu.

Reply:
Yeah, that's a good idea. Otherwise, this will come up again if changed by the user.

Reply:
It works great but I realized, doing this ended up making the dropdown list change its positioning when it is a child in a tab component

Reply:
> It works great but I realized, doing this ended up making the dropdown list change its positioning when it is a child in a tab component

I think that might be a separate issue.